cli/cli_info.cpp

```diff
@@ -119,6 +127,9 @@ command(int argc, char *argv[])
             return 1;
         }
 
+	auto &properties = p.getProperties();
+	std::string processName = properties.find("process.name")->second;
```

Comment:
This still looks incorrect to me.

Here `properties` is a `std::map`, and `std::map::find` returns `end` iterator when nothing is found, and per https://en.cppreference.com/w/cpp/container/map/end  _"attempting to access it results in undefined behavior."_

Please use the standard `std::map::find` pattern for this.  That is, something along the lines of:
```c++
auto & it = properties.find("process.name");
if (it == properties.end()) {
    std::cout << "none";    
} else {
    std::cout << '\"' << it->second << '\"';
}
```